### PR TITLE
RDM-9012 - Update V2 Swagger documentation for getStartEventTriggerUsingGET

### PR DESCRIPTION
**Pull request by sullivangareth**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9012

### Change description ###

Updated V2 swagger documentation for getStartEventTriggerUsingGET

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -42,7 +42,10 @@ def secrets = [
         secret('ccd-befta-citizen-3-pwd', 'CCD_BEFTA_CITIZEN_3_PWD'),
         secret('ccd-befta-solicitor-3-pwd', 'CCD_BEFTA_SOLICITOR_3_PWD'),
         secret('ccd-befta-caseworker-1-no-profile-pwd', 'CCD_BEFTA_CASEWORKER_1_NO_PROFILE_PWD'),
-        secret('ccd-befta-caseworker-caa-pwd', 'CCD_BEFTA_CASEWORKER_CAA_PWD')
+        secret('ccd-befta-caseworker-caa-pwd', 'CCD_BEFTA_CASEWORKER_CAA_PWD'),
+        secret('ccd-befta-master-caseworker-pwd', 'CCD_BEFTA_MASTER_CASEWORKER_PWD'),
+        secret('ccd-befta-master-solicitor1-pwd', 'CCD_BEFTA_MASTER_SOLICITOR_1_PWD'),
+        secret('ccd-befta-master-solicitor2-pwd', 'CCD_BEFTA_MASTER_SOLICITOR_2_PWD')
```
Comment (MSancaktutar) [inline Jenkinsfile_nightly:48]: couldn't quite get these.
Why would these lines be showing as different?
Did we add them as part of this ticket?

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -59,9 +62,44 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
...
+    disableLegacyDeployment()
+
+    after('test') {
+        // hmcts/cnp-jenkins-library may fail to copy artifacts after checkstyle error so repeat command (see /src/uk/gov/hmcts/contino/GradleBuilder.groovy)
+        steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/checkstyle/*.html'
+    }
+
+    after('functionalTest:preview') {
+        steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/target/cucumber/**/*'
+    }
```
Comment (MSancaktutar) [inline Jenkinsfile_nightly:99]: couldn't quite get these.
Why would the lines on this file be showing as different?
Did we add them as part of this ticket?

File: Jenkinsfile_nightly (hunk 1 above)
Comment (MSancaktutar) [reply Jenkinsfile_nightly:48]: Well, I can appreciate if the above lines may be required for nightly builds, but that should come under another ticket and PR.

File: Jenkinsfile_nightly (hunk 2 above)
Comment (MSancaktutar) [reply Jenkinsfile_nightly:99]: Well, I can appreciate if the above lines may be required for nightly builds, but that should come under another ticket and PR.

Comment (MSancaktutar) [reply Jenkinsfile_nightly]: reverted irrelevant commits from this branch.

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by hemantt — APPROVED** [review]

**Pull request merged**